forms-flow-admin/src/index.tsx

```diff
@@ -27,32 +28,58 @@ const Admin = React.memo(({ props }: any) => {
   const [roleCount, setRoleCount] = React.useState();
   const [userCount, setUserCount] = React.useState();
   const [isAdmin, setIsAdmin] = React.useState(false);
+  const [tenant, setTenant] = React.useState({});
+  const [error, setError] = React.useState();
 
-  let tenantKey = localStorage.getItem("tenantKey");
   React.useEffect(() => {
     publish("ES_ROUTE", { pathname: `${BASE_ROUTE}admin` });
   }, []);
-  //const tenantKey = tenant['tenantId'];
-  // const tenantKey = tenant?.tenantId;
+
+  let clientId = KEYCLOAK_CLIENT;
+  const url = window.location.href;
+  const url_array = url.split('/');
+  const tenantKey = url_array[4];
+  clientId = MULTITENANCY_ENABLED ? `${tenantKey}` : KEYCLOAK_CLIENT;
+
+
   React.useEffect(() => {
     if (!isAuth) {
       let instance = KeycloakService.getInstance(
         KEYCLOAK_URL_AUTH,
         KEYCLOAK_URL_REALM,
         KEYCLOAK_CLIENT,
-        tenantKey
+        clientId,
```

Comment:
according to the definition, this should be tenantKey, and if not present default client will be used so better to pass this argument as tenant key if tenant-based else None